core/App/screens/CredentialDetails.tsx

```diff
@@ -4,44 +4,42 @@ import { CredentialRecord } from '@aries-framework/core'
 import { useCredentialById } from '@aries-framework/react-hooks'
 import React, { useEffect, useState } from 'react'
 import { useTranslation } from 'react-i18next'
-import { StyleSheet, Text, TouchableOpacity, View } from 'react-native'
+import { View } from 'react-native'
 import Toast from 'react-native-toast-message'
 
 import CredentialCard from '../components/misc/CredentialCard'
 import InfoBox, { InfoBoxType } from '../components/misc/InfoBox'
 import Record from '../components/record/Record'
 import { ToastType } from '../components/toast/BaseToast'
+import { DispatchAction } from '../contexts/reducers/store'
 import { useStore } from '../contexts/store'
-import { useTheme } from '../contexts/theme'
 import { CredentialStackParams, Screens } from '../types/navigators'
-import { testIdWithKey } from '../utils/testable'
 
 type CredentialDetailsProps = StackScreenProps<CredentialStackParams, Screens.CredentialDetails>
 
 const CredentialDetails: React.FC<CredentialDetailsProps> = ({ navigation, route }) => {
   const { t } = useTranslation()
-  const { ColorPallet, TextTheme } = useTheme()
-  const [state] = useStore()
-  const { revoked } = state.credential
+  const [state, dispatch] = useStore()
+  const { revoked, revokedMessageDismissed } = state.credential
   const [isRevoked, setIsRevoked] = useState<boolean>(false)
-  const [isCredentialRevokedMessageHidden, setIsCredentialRevokedMessageHidden] = useState<boolean>(false)
-  const styles = StyleSheet.create({
-    headerText: {
-      ...TextTheme.normal,
-    },
-    footerText: {
-      ...TextTheme.normal,
-      paddingTop: 16,
-    },
-    linkContainer: {
-      minHeight: TextTheme.normal.fontSize,
-      paddingVertical: 2,
-    },
-    link: {
-      ...TextTheme.normal,
-      color: ColorPallet.brand.link,
-    },
-  })
+  const [isRevokedMessageHidden, setIsRevokedMessageHidden] = useState<boolean>(false)
+  // const styles = StyleSheet.create({
```

Comment:
Needed?

Reply:
It will be once the `Remove from wallet` link is added back. I didn't want to lose context of the styling. Although we could just look back in the Git history.